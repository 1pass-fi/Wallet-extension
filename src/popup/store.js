import { composeWithDevTools } from 'redux-devtools-extension'
import { createStore, applyMiddleware } from 'redux'
import thunk from 'redux-thunk'

import rootReducer from './reducers/index'

const store = createStore(
  rootReducer,
  composeWithDevTools(applyMiddleware(thunk))
)

export default store
