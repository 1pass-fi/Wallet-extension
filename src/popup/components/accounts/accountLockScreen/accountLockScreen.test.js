import '@babel/polyfill'
import { render } from '@testing-library/react'
import '@testing-library/jest-dom/extend-expect'
import { Provider } from 'react-redux'
import store from 'popup/store'
import React from 'react'

import AccountLockScreen from '.'

import { BrowserRouter as Router } from 'react-router-dom'

describe('Test for AccountLockScreen component', () => {
  describe('Render without crashing', () => {
    it('renders correctly', () => {
      const { container } = render(<Router><Provider store={store}><AccountLockScreen /></Provider></Router>)
      expect(container).toMatchSnapshot()
    })
  })
})
