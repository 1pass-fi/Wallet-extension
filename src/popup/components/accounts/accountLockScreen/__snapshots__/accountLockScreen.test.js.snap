// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test for AccountLockScreen component Render without crashing renders correctly 1`] = `
<div>
  <div
    class="unlock-screen"
  >
    <div
      class="screen-header"
    >
      <svg
        class="koi-icon"
        height="130"
        width="130"
      />
      <div
        class="title"
      >
        Koi Wallet
      </div>
    </div>
    <div
      class="screen-content"
    >
      <div
        class="welcome"
      >
        Welcome back
      </div>
      <div
        class="input-field input-password"
      >
        <div
          class="label"
        >
          Enter your password
        </div>
        <div
          class="input-wrapper"
        >
          <input
            class="input"
            name=""
            placeholder=""
            type="password"
            value=""
          />
          <svg
            class="toggle-display"
          />
        </div>
      </div>
      <button
        class="button-shared button-filled unlock-button"
      >
        Unlock
      </button>
      <div
        class="seed-phrase"
      >
        Import my account using myÂ 
        <a
          class="link-to-seed-phrase"
          href="/account/login/phrase"
        >
          seed phrase
        </a>
        .
      </div>
    </div>
  </div>
</div>
`;
